src/net40/Radical.Windows.Presentation/Services/ConventionsHanlder.cs

```diff
@@ -33,6 +34,8 @@ public ConventionsHanlder(IMessageBroker broker, IReleaseComponents releaser, Bo
         {
             this.broker = broker;
             this.releaser = releaser;
+            this.releaserSupportsDispose = releaser.GetType().IsAttributeDefined<SupportComponentDisposeAttribute>();
```

Comment:
Yes. It's a Singleton and cannot change at runtime, so we can check it once at startup